oss-internship-2020/curl/tests/test_utils.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TESTS_H
+#define TESTS_H
+
+#include <future>
+
+#include "../sandbox.h"
+#include "curl_sapi.sapi.h"
+#include "gtest/gtest.h"
+#include "sandboxed_api/util/flag.h"
+#include "sandboxed_api/util/status_matchers.h"
+
+// Helper class that can be used to test Curl Sandboxed
+class CurlTestUtils {
+ protected:
+  struct MemoryStruct {
+    char* memory;
+    size_t size;
+  };
+
+  // Initialize and set up the curl handle
+  void curl_test_set_up();
```

Comment:
Please rename these functions according to style guide: https://google.github.io/styleguide/cppguide.html#Function_Names